react/features/gifs/functions.web.ts

```diff
@@ -11,7 +11,12 @@ export * from './function.any';
  */
 export function isGifsMenuOpen(state: IReduxState) {
     const overflowDrawer = showOverflowDrawer(state);
-    const { drawerVisible, menuOpen } = state['features/gifs'];
+    const { overflowMenuVisible } = state['features/toolbox'];
```

Comment:
Why do we need to check if the overflow menu is visible? Isn't it enough to check if the gif menu is open?

Reply:
I was actually doing the same as we do for the drawer. 

On the second look I think we don't need checks at all. I changed it to check only for `menuOpen` even for the drawer case. The only side effect is if you change layouts and we go from reactions displayed in drawer to overflow menu to the main toolbar the state of the gif menu will be preserved which IMHO is not bad and this is kind of a corner case.

Good catch! This simplifies the logic a lot.